GameOver.cs

```diff
@@ -0,0 +1,21 @@
+using Godot;
+using System;
+
+public partial class GameOver : Node2D
+{
+	private void _on_button_pressed()
```

Comment:
Even if it's obvious, could you add the specification for this function?

Reply:
ok